checkout-extension/README.md.liquid

```diff
@@ -0,0 +1,88 @@
+# Checkout UI Extension
+
+Checkout UI extensions let app developers build custom functionality that merchants can install at defined points in the checkout flow. You can learn more about checkout UI extensions in Shopify’s [developer documentation](https://shopify.dev/api/checkout-extensions/checkout).
+
+
+## Prerequisites
+Make sure to complete [all the prerequisites](https://shopify.dev/apps/checkout/delivery-instructions/getting-started#requirements) from any tutorials to develop a Checkout UI extension.
+
+## Tutorials
+The best way to get started is to follow some of the available Extension Tutorials:
+
+For creating an extension:
+* [Enable extended delivery instructions](https://shopify.dev/apps/checkout/delivery-instructions)
+* [Adding product offer recommendations](https://shopify.dev/apps/checkout/product-offers)
+* [Creating a custom banner](https://shopify.dev/apps/checkout/custom-banners)
+
+To add specific features to an extension
+* [Adding field validation](https://shopify.dev/apps/checkout/validation)
+* [Localizing an extension](https://shopify.dev/apps/checkout/localize-ui-extensions)
+
+## Getting Started
+Initially, your extension will have the following files:
+
+```
+└── my-app
+  └── extensions
+    └── my-checkout-ui-extension
+        ├── src
+        │   └── CheckoutDynamicRender.{{ srcFileExtension }} The source code for the CHECKOUT:DYNAMIC::RENDER extension point
+        ├── locales
+        │   ├── en.default.json // The default locale for the checkout UI extension. Shared across all extension points.
+        │   └── fr.json // The locale file for non-regional French translations. Shared across all extension points.
+        └── shopify.ui.extension.toml // The config file for the checkout UI extension
+
+```
+
+You can customize your new extension by editing the code in the `src/CheckoutDynamicRender.{{ srcFileExtension }}` file.
+
+> By default, your extension is configured to target the `Checkout::Dynamic::Render` [extension point](https://shopify.dev/api/checkout-extensions/checkout#extension-points). This extension point does not have a single location in the checkout where it will appear; instead, a merchant installing your extension will configure where *they* want your extension to show up.
+> If you are building an extension that is tied to existing UI element in the checkout, such as the cart lines or shipping method, you can change the extension point so that your UI extension will render in the correct location. Check out the list of [all available extension points](https://shopify.dev/api/checkout-extensions/checkout#extension-points) to get some inspiration for the kinds of content you can provide with checkout UI extensions.
```

Comment:
```suggestion
> If you are building an extension that is tied to existing UI element in the checkout, such as the cart lines or shipping method, you can change the extension point so that your UI extension will render in the correct location. Check out the list of [all available extension points](https://shopify.dev/api/checkout-extensions/checkout#extension-targets) to get some inspiration for the kinds of content you can provide with checkout UI extensions.
```